web/app/configs/lang.json

```diff
@@ -376,5 +376,68 @@
         "TR": "Turkiet",
         "UA": "Ukraina"
     }
+  },
+  "de": {
+    "wind": "Wind",
+    "solar": "Solar",
+    "hydro": "Wasser",
+    "hydro storage": "Pumpspeicher",
+    "biomass": "Biomasse",
+    "nuclear": "Atomkraft",
+    "geothermal": "Geothermie",
+    "gas": "Gas",
+    "coal": "Kohle",
+    "oil": "Öl",
+    "unknown": "unbekannt",
+    "exportto": "des Exports zu",
```

Comment:
unbekannt ist ok

For "Oil", it's to be understood as "Erdöl" I believe. But maybe "Öl" is what German use ?

For Export to, I'm not sure it is still used, I did not find it in the map.  But your translation seems ok.